addons/common/fnc_randomizeFacewear.sqf

```diff
@@ -0,0 +1,43 @@
+#include "script_component.hpp"
+/* ----------------------------------------------------------------------------
+Function: CBA_fnc_randomizeFacewear
+
+Description:
+    Add config defined weighted random facewear to unit.
+    Uses same config as headgearList[].
+
+Parameters:
+    _unit - unit <OBJECT>
+
+Returns:
+    true on success, false on error <BOOLEAN>
+
+Examples:
+    (begin example)
+        [unit] call CBA_fnc_randomizeFacewear;
+    (end)
+
+Author:
+    commy2
+---------------------------------------------------------------------------- */
+
+params [["_unit", objNull]];
+
+if (isNull _unit) exitWith {
+    TRACE_1("unit is null",_unit);
```

Comment:
WARNING_1 macro instead? (has to use %1)

Reply:
Other things just use a TRACE, but I guess they are meant to fail silently.